sql/core/src/main/scala/org/apache/spark/sql/execution/columnar/InMemoryTableScanExec.scala

```diff
@@ -36,6 +36,15 @@ case class InMemoryTableScanExec(
     @transient relation: InMemoryRelation)
   extends LeafExecNode with ColumnarBatchScan {
 
+  override val nodeName: String = {
+    relation.cacheBuilder.tableName match {
+      case Some(tableName) =>
+        s"Scan in memory table ${tableName.replace("`", "")}"
```

Comment:
@wangyum . Could you improve this more because we allow the backtick in `tableName`?
```scala
scala> sql("create temporary view ab as select 1")

scala> sql("create temporary view ```a``b``` as select 100000")

scala> sql("show tables").show
+--------+---------+-----------+
|database|tableName|isTemporary|
+--------+---------+-----------+
|        |    `a`b`|       true|
|        |       ab|       true|
+--------+---------+-----------+
```

Reply:
How about `relation.cacheBuilder.cachedColumnBuffers.name`. Consistent with the `StorageTab`?
```scala
  override val nodeName: String = {
    relation.cacheBuilder.tableName match {
      case Some(_) =>
        "Scan " + relation.cacheBuilder.cachedColumnBuffers.name
      case _ =>
        super.nodeName
    }
  }
``` 
![image](https://user-images.githubusercontent.com/5399861/57265772-c6e31480-70ab-11e9-9517-089b43e26ac5.png)

![image](https://user-images.githubusercontent.com/5399861/57265991-9d76b880-70ac-11e9-9b52-f0c0a17ce1b8.png)

Reply:
+1. Sounds nice to me. Thanks.